site2/docs/reference-configuration-pulsar-proxy.md

```diff
@@ -0,0 +1,934 @@
+# Pulsar proxy
+## Required
```

Comment:
If there is no `required` configuration, can you add an explanation? Something like `All configurations for Pulsar proxy are optional`?